lighthouse-core/test/report/html/renderer/performance-category-renderer-test.js

```diff
@@ -26,15 +27,14 @@ const TEMPLATE_FILE = fs.readFileSync(__dirname +
 describe('PerfCategoryRenderer', () => {
   let category;
   let renderer;
+  global.CategoryRenderer = CategoryRenderer;
+  const PerformanceCategoryRenderer =
+      require('../../../../report/html/renderer/performance-category-renderer.js');
 
   beforeAll(() => {
     global.URL = URL;
     global.Util = Util;
     global.CriticalRequestChainRenderer = CriticalRequestChainRenderer;
-    global.CategoryRenderer = CategoryRenderer;
```

Comment:
why move?

Reply:
`PerformanceCategoryRenderer` has to be in a higher scope (for the new suite)
and `PerformanceCategoryRenderer` extends this one, so the require will fail if this isn't done beforehand